### ISSUE TITLE
KeyErro module

### ISSUE THREAD

**Issue by alexauvray** [Battleman/PyMailExtract#1]
```
Page 1
Traceback (most recent call last):
  File "Pymailextract.py", line 223, in <module>
    main()
  File "Pymailextract.py", line 209, in main
    token = get_message_list()
  File "Pymailextract.py", line 176, in get_message_list
    service = get_service()
  File "Pymailextract.py", line 40, in get_service
    creds = store.get()
  File "/usr/local/lib/python3.7/site-packages/oauth2client/client.py", line 407, in get
    return self.locked_get()
  File "/usr/local/lib/python3.7/site-packages/oauth2client/file.py", line 54, in locked_get
    credentials = client.Credentials.new_from_json(content)
  File "/usr/local/lib/python3.7/site-packages/oauth2client/client.py", line 302, in new_from_json
    module_name = data['_module']
KeyError: '_module'
```

**Comment by Battleman** [comment]
Thanks, will look into that

**Comment by alexauvray** [comment]
No worries. Would love to make it work. Does it work for you with the new Gmail ?

**Comment by alexauvray** [comment]
It might be possible that the client_secret.json file which is downloaded from Google API console should be placed in the same directory as the python quickstart.py file and not inside ~/.credentials directory.

**Comment by Battleman** [comment]
It should be fixed in #3. PR was a bit dirty, but basically using new API specifications. Closing it. let me know if not fixed, will reopen
